funding_csv_to_hash.rb

```diff
@@ -0,0 +1,45 @@
+require 'csv'
```

Comment:
Simple script to write the CSV data into a hash which is then stored in a text fie.